variantlib/commands/config/show.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import sys
+
+import tomlkit
+import tomllib
```

Comment:
You need a fallback for Py < 3.11. Or just use `tomlkit` to read files too.

Reply:
Done